app/src/main/java/com/example/concordia_campus_guide/Global/User.java

```diff
@@ -1,30 +1,30 @@
-package com.example.concordia_campus_guide.Global;
-
-import com.example.concordia_campus_guide.Models.Profile;
-
-import java.util.List;
-
-
-//@Entity (tableName = "users")
-public class User {
-
-    //@Ignore
-    public static User instance;
-
-    //@ColumnInfo (name ="id")
-    private long id;
-
-
-    private List schedule;
-
-    //@Embedded
-    private Profile profile;
-
-    public User() {}
-
-    private void fetchSchedule() {}
-
-    public User getInstance() {
-        return instance;
-    }
-}
+//package com.example.concordia_campus_guide.Global;
```

Comment:
Can we add a TODO and create a task to fix this test?

Reply:
ok this is not a test. It is the user class that we have implemented. I commented it out because i was trying something. I will uncomment it before i merge.